client/post-editor/restore-post-dialog.jsx

```diff
@@ -26,7 +26,7 @@ class EditorRestorePostDialog extends Component {
 		onRestore: PropTypes.func,
 		isAutosave: PropTypes.bool,
 		postType: PropTypes.string,
-		userCanDeletePost: PropTypes.bool,
+		canUserDeletePost: PropTypes.bool,
```

Comment:
Here you ended up just renaming `userCanDeletePost` to `canUserDeletePost`, which is not strictly needed for anything. Consider reverting the change if it was not 100% intentional.

Reply:
Yeah, good point. I'll do that.

Reply:
Actually, it's in two commits now. I would have to fiddle with the history and the rename is not bad. I'll keep it.